HomeWorks/HW11/IMotorcycleRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace HW11
+{
+    interface IMotorcycleRepository
+    {
+        void CreateMotorcycle(Motorcycle obj);
+
+        List<Motorcycle> GetMotorcycle();
+
+        Motorcycle GetMotorcycle(long id);
+
+        void UpdateMotorcycle(long id);
```

Comment:
usually, we need to put the object instead of id

`void UpdateMotorcycle(Motorcycle obj);`